pkg/skel/skel.go

```diff
@@ -93,14 +93,20 @@ func PluginMain(cmdAdd, cmdDel func(_ *CmdArgs) error) {
 	}
 
 	if err != nil {
+		if e, ok := err.(*plugin.Error); ok {
+			// don't wrap Error in Error
+			e.Print()
```

Comment:
yeah but what you gonna do. it's already exiting with non-zero status and failed to write the error. i'll add a log to stderr.